portfolio/src/main/webapp/information.html

```diff
@@ -107,59 +107,39 @@ <h1 style="text-align: center; font-size: 30px">Experience</h1>
       <h2 style="text-align: left center">Thomas Pena</h2>
       <a>
-        <p style="right: 25px; position: absolute; font-size: 30px">
-          [super cool description goes here] University of Florida UF College of
-          Engineering in Gainesville, FL Graduation: May 2023<br />
-          Bachelor of Science: Computer Science<br />
-          Society of Hispanic Professional Engineers (SHPE UF)<br />
-          Triathlon Club at the University of Florida
+        <p style="right: 30px; position: absolute; font-size: 30px">
+             Hi! I'm Thomas. I'm from Bogota, Colombia
+          and I'm currently studying at the University of Florida.<br />
+          Technology and innovation has always fascinated me that is why I'm 
+          majoring in Computer Science!<br />
+          Some fun facts about me: <br />• Oreo lover <br />
+          • I speak three languages<br />
+          • Currently training to run a marathon by the end of this year!
         </p>
         <img src="/images/Pena, Thomas.jpg" class="profile" />
       </a>
     </div>
 
     <div class="blog">
       <h1 style="text-align: center; font-size: 30px; color: whitesmoke">
-        Blog
+        Chat
       </h1>
-      <p style = "color:whitesmoke">Click this button to get a question.</p>
-      <button onclick="showRandomQuestion();">
+      <div>
+      <div class="centerButton">
+      <button   onclick="showRandomQuestion();">
       Get a question
       </button>
-      <p style = "color:whitesmoke" id="question-container"></p>
-      <p style = "color:whitesmoke">Click <a href="/chat">here</a> to chat with me!</p>
-      <textarea>
-        Welcome to my Blog :)
-
-        Today I have a challenge for you. 
-        Find the issues in the following code:
-
-            public Integer find(Integer[] arr) {
-                max = Integer.MIN_VALUE;
-                int secondMax = Integer.MIN_VALUE
-
-                for (int i = 0; i <= arr.length, i++) {
-                    if (arr[i] > max) {
-                    secondMax == max;
-                    max = arr[i];
-                    }
-
-                    if (arr[i] < max & arr[i] > secondMax) {
-                    secondMax = arr[i];
-                    }
-                }
-            return second;
-            }
-      </textarea>
-    <div>
-        <button onclick="onClickShowBlogSolution()">solution</button>
-        <a id="blog-solution" style="display: none">
-          <img
-            src="/images/blogChallengeSolution.JPG"
-            style="background-position: right top"
-          />
-        </a>
       </div>
+      <p style = "text-align: center;color:whitesmoke" id="question-container"></p>
+      <form style="top: 20%" action="/form-handler" method="POST">
+      <textarea style="top: 30%" name="text-input"></textarea>
+      <br/>
+      <div style="top: 60%" class="centerButton">
+      <input type="submit" />
+      </div>
+      </form>
+
+         
```

Comment:
Looks like you have some extra spaces here, if you need new lines, please use <br>, otherwise please remove the extra spaces.